src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const minNumDayDiscount = 3;
+  const maxNumDayDiscount = 7;
+
+  if (days >= maxNumDayDiscount) {
+    return days * dailyPrice - maxDiscount;
+  }
+
+  if (days >= minNumDayDiscount && days < maxNumDayDiscount) {
```

Comment:
Second condition is redundant.
if `days >= maxNumDayDiscount` is false, `days < maxNumDayDiscount` is always true.